src/world/components/lib/getRandomTransition.js

```diff
@@ -0,0 +1,7 @@
+const matchTransitionFor = require('./matchTransitionFor');
+
+module.exports = function getRandomTransition(world) {
+  const moveableSet = world.things.filter(thing => thing.locations.length > 1);
+  const randomThing = moveableSet[Math.floor(Math.random() * moveableSet.length)];
+  return randomThing && [matchTransitionFor(randomThing, world.numRules, world.rules), randomThing];
+}
```

Comment:
moveableSet is the set of every thing that has more than one potential location. Because there are multiple possible locations for that thing we assume it is eligible for location changes.

Then we pick a random "moveableThing" and try to find a transition rule for it with the function matchTransitionFor. This is part of the current "run story graph and get a random story" paradigm that I am starting to question.